quick_start.rst

```diff
@@ -605,9 +617,9 @@ do with normal Linux commands.
 
 .. _build-images-from-scratch:
 
-***************************
- Build images from scratch
-***************************
+*****************************
+ Building images from scratch
```

Comment:
Can remove the leading space here.

The most common way of writing an RST heading has the overline/underline *exactly* the same length as the title text.

Having the overline/underline 1 char before, and 1 char after the title test is from an outdated linting/formatting tool that was applied at some point.